Tasks/NodeTool/nodetool.ts

```diff
@@ -17,7 +17,7 @@ async function run() {
         await getNode(versionSpec, onlyLTS);
     }
     catch (error) {
-        console.error('ERR:' + error.message);
+        taskLib.setResult(taskLib.TaskResult.Failed, error.message);
```

Comment:
it took me a while to realize why errors weren't failing the task. i thought we had a bigger problem :)